lib/sub-process.ts

```diff
@@ -1,7 +1,12 @@
-import * as childProcess from 'child_process';
+import { spawn, spawnSync, SpawnOptions } from 'child_process';
 
-function makeSpawnOptions(options) {
-  const spawnOptions: childProcess.SpawnOptions = { shell: true };
+interface ProcessOptions {
+  cwd?: string;
+  env?: object;
```

Comment:
I think we should never type as `object`. If these are env properties, will all keys be strings? What about the values?

Could probably be `Record<string, string>`